docs/development.md

```diff
@@ -37,8 +37,6 @@ These can be provided on the command line with `go test` or with program argumen
 
 - `--e2e` - If true, run end-to-end tests. (required to run the e2e tests)
 - `--debug` - If true, do not destroy cluster and clean up temporary directory after test.
-- `--image-prefix` - The prefix to use for Docker images. Defaults to the local Docker registry. Omit the trailing slash.
```

Comment:
One concern about removing `--image-prefix` and `--image-tag` is that it is not that flexible to test with different images.

Previously, it was pretty easy to test with a particular image by setting  `--image-tag`. If we want to test with a different image, we just update the flag. Now, with this approach, we have to re-generate the .output directory. If we want to go back and forth between different images, we have to version the .output directory and rename it. Is there a better way to do so?

Reply:
My intent for this use case is that the developer just runs `make build-manifests IMAGE_TAG=<TAG>` if they want to switch between manifests for test runs. This is fairly light weight and runs quickly for swapping back and forth.

Alternatively we could pass the flags into the tests and then the tests shell back to `make build-manifests` to construct the manifests, although I think I prefer the simplicity of just running `go test --e2e` and it will use whichever manifests are on the filesystem

Reply:
Using `make build-manifests IMAGE_TAG=<TAG>` sounds good. Make sure we document the process for posterity.

Reply:
Added some documentation, PTAL